apps/wallet/src/ui/app/shared/faucet/FaucetRequestButton.tsx

```diff
@@ -46,7 +46,7 @@ function FaucetRequestButton({ variant = 'primary', size = 'narrow' }: FaucetReq
 			onClick={() => {
 				toast.promise(mutation.mutateAsync(), {
 					loading: <FaucetMessageInfo loading />,
-					success: (totalReceived) => <FaucetMessageInfo totalReceived={totalReceived} />,
+					success: () => <FaucetMessageInfo message="Request in progress." />,
```

Comment:
Do we really want it to work this way, or do we want to just make the mutation poll the status API to know when it actually succeeds?

Reply:
I had the polling option locally until, I realized the endpoint was using GET request with body.